public/app/plugins/panel/geomap/utils/getGeoMapStyle.ts

```diff
@@ -0,0 +1,32 @@
+import { Style, Stroke, Fill } from 'ol/style';
+import { ColorDimensionConfig } from 'app/features/dimensions';
+import { GeoJSONMapperRule } from './checkFeatureMatchesStyleRule';
+
+export interface GeoMapStyle {
```

Comment:
can GeoMapStyle and GeoJSONMapperRule both be defined in types.ts?

maybe:
GeoMapStyle => FeatureStyleConfig
GeoJSONMapperRule => FeatureRuleConfig
?